tests/gold_tests/tls_hooks/tls_hooks9.test.py

```diff
@@ -39,8 +39,6 @@
     'proxy.config.diags.debug.tags': 'ssl_hook_test',
     'proxy.config.ssl.server.cert.path': '{0}'.format(ts.Variables.SSLDir),
```

Comment:
@macisasandwich If this something we pulled in as well? ( another PR to clean up C&P code :-)

Reply:
these were probably part of the original test back when it was committed. I will do another PR somwhere down the line to clean up these as well and use the test extension's functions.